model/src/main/proto/exploration.proto

```diff
@@ -322,6 +322,27 @@ message UserAnswer {
   WrittenTranslationContext written_translation_context = 6;
 }
 
+// Corresponds to a raw item selection answer that user has submitted.
+message ItemSelectionRawAnswer {
+  repeated int32 selected_indexes = 1;
+}
+
+// Corresponds to a last raw answer entered by user which is used to retain state on Config Change.
+message RawUserAnswer {
+  oneof answer_input_type {
+    // A raw answer entered by user in a text-based interaction.
+    string textual_answer = 1;
+    // An indices selected in multiple choice selection.
+    int32 multiple_choice_selection_index = 2;
```

Comment:
Is this actually being used? It seems like the selection is being stored only in ItemSelectionRawAnswer in the multiple choice case.